spaceship/lib/spaceship/tunes/tunes_client.rb

```diff
@@ -86,6 +86,12 @@ def select_team
           puts "#{i + 1}) \"#{team['contentProvider']['name']}\" (#{team['contentProvider']['contentProviderId']})"
         end
 
+        unless Spaceship::Client::UserInterface.interactive?
+          puts "Multiple teams found on iTunes Connect, Your Terminal is running in non-interactive mode! Cannot continue from here."
+          puts "Please check that you set FASTLANE_ITC_TEAM_ID or FASTLANE_ITC_TEAM_NAME to the right value."
+          raise "Multiple iTunes Connect Teams found; unable to choose, terminal not ineractive!"
```

Comment:
I don't think so, as _spaceship_ doesn't have access to FastlaneCore, including UserError